### PR TITLE
Updated UG to include incorrect user input case

### PR DESCRIPTION
**Pull request by reetmitra**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -276,7 +276,21 @@ will only show an error message (`No new changes has been made to the data`).
...
 **Note:** `undo` does not affect commands that do not modify the contact list
-(`help`, `list`, `find` etc.). 
+(`help`, `list`, `find` etc.).
+
+### Examples of Incorrect Inputs
+
+1. **Example 1: Empty Tag in Add Command**
+
+   If you enter the command `add n/John Doe p/12345678 e/johndoe@example.com r/Student a/PGPR c/CS2103T t/`, 
+   you will see an error message because the `t/` tag is empty.
```
Comment (Hibeom0929) [inline docs/UserGuide.md:286]: It would be better to show which error message.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -276,7 +276,21 @@ will only show an error message (`No new changes has been made to the data`).
...
+1. **Example 1: Empty Tag in Add Command**
+
+   If you enter the command `add n/John Doe p/12345678 e/johndoe@example.com r/Student a/PGPR c/CS2103T t/`, 
+   you will see an error message because the `t/` tag is empty. 
+   The correct command should be `add n/John Doe p/12345678 e/johndoe@example.com r/Student a/PGPR c/CS2103T t/friend`.
+
+2. **Example 2: Incorrect Usage of Copy Command**
+
+   If you want to copy the email of the first person in your contact list, you would use the command `copy 1`. 
+   If you enter `copy` without specifying the index, you will see an error message because the index is missing.
```
Comment (Hibeom0929) [inline docs/UserGuide.md:292]: Same thing here.

**Review by Hibeom0929 — COMMENTED** [review]
Also you can consider to add screenshot for reference.

File: docs/UserGuide.md (hunk 1 above)
Comment (Hibeom0929) [reply docs/UserGuide.md:286]: I think can do both. But writing down the error message is sth necessary, and providing a screenshot is optional. But you can do what you think right, just make sure to show a clear result when user typed wrongly.

**Comment by E0735389** [comment]
Why do we want to do this? Does it make the user guide easier to understand?

**Comment by E0735389** [comment]
Have you loading at the generated website locally? Do the images render correctly?

**Comment by Hibeom0929** [comment]
> Why do we want to do this? Does it make the user guide easier to understand?

The idea came from CS2101. But I'm not sure if we want to this, we should list all the error messages

**Comment by Hibeom0929** [comment]
Sry to confuse you too much but I think just writing down instead of a screenshot is preferred as the screenshot can't show the full error message.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -276,7 +276,23 @@ will only show an error message (`No new changes has been made to the data`).
...
+   you will see an error message because the `t/` tag is empty.
+   ![Empty Tag Error](images/EmptyTagError.png)
+   The correct command should be `add n/John Doe p/12345678 e/johndoe@example.com r/Student a/PGPR c/CS2103T t/friend`.
+
+2. **Example 2: Incorrect Usage of Copy Command**
+
+   If you want to copy the email of the first person in your contact list, you would use the command `copy 1`. 
+   If you enter `copy` without specifying the index, you will see an error message because the index is missing.
+   ![Copy Error](images/CopyError.png)
+   The correct command should include the index of the person in the displayed contact list, like `copy 1`.
```
Comment (Hibeom0929) [inline docs/UserGuide.md:295]: This line is basically saying the same thing with the line 292 " If you want to copy the email of the first person in your contact list, you would use the command `copy 1`."

Comment (reetmitra) [reply docs/UserGuide.md:295]: okay this one i can edit to make it sound a little less redundant

**Comment by reetmitra** [comment]
wait so you don't want me to include the screenshot and write out the error message instead?

**Pull request closed without merging**